src/ui/components/Rating/index.js

```diff
@@ -68,8 +68,13 @@ export class RatingBase extends React.Component {
         id: `Rating-rating-${thisRating}`,
         key: `rating-${thisRating}`,
         ref: (ref) => { this.ratingElements[thisRating] = ref; },
+        title: !readOnly ? i18n.sprintf(i18n.gettext(`Rate this add-on %(thisRating)s out of 5.`), { thisRating }) : null,
```

Comment:
I think this `title` setting code would be a bit cleaner if you put both cases into a function, which would return the correct title, instead of setting a default title here, and then conditionally setting it again below.

Also, please try to keep the line length at a maximum of 80 characters.